block/internal/submitting/da_submitter.go

```diff
@@ -25,7 +25,7 @@ const (
 	initialBackoff               = 100 * time.Millisecond
 	defaultGasPrice              = 0.0
 	defaultGasMultiplier         = 1.0
-	defaultMaxBlobSize           = 2 * 1024 * 1024 // 2MB fallback blob size limit
+	defaultMaxBlobSize           = 1.5 * 1024 * 1024 // 1.5MB fallback blob size limit
```

Comment:
if we merge this, we need to decrease it here too: https://github.com/evstack/ev-node/blob/main/da/jsonrpc/internal/consts.go#L20